tests/Backend/SOX/MergeRequestsTest.php

```diff
@@ -672,11 +676,11 @@ public function testMrWithConflictCantBeMergedButAfterResetCan(): void
 
         $devBranchComponents->resetToDefault($componentId, $configurationId);
 
-        $actualIdentifierInMain = $components->getConfiguration($componentId, $configurationId);
-        $actualIdentifierInBranch = $devBranchComponents->getConfiguration($componentId, $configurationId);
+        $actualIdentifierInMain = $components->getConfiguration($componentId, $configurationId)['currentVersion']['versionIdentifier'];
+        $actualIdentifierInBranch = $devBranchComponents->getConfiguration($componentId, $configurationId)['currentVersion']['versionIdentifier'];
```

Comment:
Neměla by tady být taky v1?

Reply:
tam sa testuje pak něco jiného, já sem enom upravil ten přístup na pole protože název té proměnné kecal.

Reply:
Mě přijde, že se testuje, že nově to co předtím nefungovalo (v1 != default) teď funguje po tom cos to resetnul. Ale OK. Tady mi to asi žílu netrhá.